lib/src/utils/resolveRichTextToNodes.ts

```diff
@@ -251,9 +258,11 @@ export const resolveNode = (
       content: content.map((node) => {
         // skip rendering p tag inside li
         if (node.type === "paragraph") {
-          return {
-            content: node.content.map((node) => resolveNode(node, options)),
-          };
+          return node.content
+            ? {
+                content: node.content.map((node) => resolveNode(node, options)),
+              }
+            : "";
```

Comment:
Having in mind the other comment about `[""]` case, you may go with the following instead:

```
return {
  content: node.content?.map((node) => resolveNode(node, options)) || "",
};
```